packages/salesforcedx-utils-vscode/src/services/o11yService.ts

```diff
@@ -28,14 +28,14 @@
 import { z } from 'zod';
 import { loadO11yModules } from '../telemetry/utils/o11yLoader';
 
+const O11Y_UPLOAD_THRESHOLD_BYTES = 50_000;
 export class O11yService {
-  O11Y_UPLOAD_THRESHOLD_BYTES = 50_000;
-  o11yUploadEndpoint: string | undefined;
-  instrumentation: Instrumentation;
-  _instrApp: InstrumentedAppMethods;
-  protoEncoderFunc: ProtoEncoderFuncType | null = null;
-  a4dO11ySchema: unknown;
-  readonly environment: Record<string, string> = {};
+  private o11yUploadEndpoint: string | undefined;
```

Comment:
olly stuff is a prime example of why these rules will help...there's stuff that was relying on "default public" used only for tests.

I want people to think a bit more about what they expose and least-privilege class stuff.